src/controller.rs

```diff
@@ -201,6 +202,10 @@ impl Controller for FoxBox {
     fn get_upnp_manager(&self) -> Arc<UpnpManager> {
         self.upnp.clone()
     }
+
+    fn get_config(&self) -> Arc<ConfigService> {
```

Comment:
This part is not that great, I concede. However, getting the upnp address from a config file is the last thing blocking the integration tests. So I'd prefer to change `get_config()` in a follow up patch. What do you think @cr ?